components/empty-view/src/styles.ts

```diff
@@ -48,3 +54,12 @@ export const useStyles = makeStyles({
     verticalAlign: "middle",
   },
 });
+
+export function useScreenStylesStyles({
+  className,
+}: HtmlDivAttributesRestProps) {
+  const styles = useStyles();
+  const containerStyle = mergeClasses(styles.container, className);
+
+  return { styles, containerStyle };
+}
```

Comment:
If we export both `useStyles` and `useScreenStylesStyles`, you don't need to return styles from `useScreenStylesStyles`. 
Also I would rename this to `useContainerStyle` and return just `mergeClasses(styles.container, className)` instead.

Reply:
should be fixed now